test/externalTests.sh

```diff
@@ -150,12 +150,13 @@ function force_solc_settings
   local evmVersion="$3"
 
   printLog "Forcing solc settings..."
+  echo "-------------------------------------"
   echo "Config file: $config_file"
   echo "Optimizer settings: $settings"
   echo "EVM version: $evmVersion"
-  echo ""
+  echo "-------------------------------------"
 
-  echo "module.exports['compilers']['solc']['settings'] = { optimizer: $settings, evmVersion: \"$evmVersion\" };" >> "$config_file"
```

Comment:
Actually I would add both. It might be a version problem.

Reply:
Sounds good!